src/openzaak/components/catalogi/admin/admin_views.py

```diff
@@ -54,9 +71,12 @@ def post(self, request, *args, **kwargs):
             else:
                 try:
                     with transaction.atomic():
-                        import_zaaktype_for_catalogus(
-                            catalogus_pk, request.session["file_content"], {}, {}
-                        )
+                        with requests_cache_enabled(
+                            "import", backend=DjangoRequestsCache()
+                        ):
+                            import_zaaktype_for_catalogus(
+                                catalogus_pk, request.session["file_content"], {}, {}
+                            )
```

Comment:
shouldn't we apply that context manager inside the `import_zaaktype_for_catalogus` function instead, or apply it as a decorator?